src/main/java/hudson/plugins/pmd/PmdPublisher.java

```diff
@@ -25,14 +24,20 @@
  * @author Ulli Hafner
  */
 public class PmdPublisher extends HealthAwarePublisher {
+
     private static final long serialVersionUID = 6711252664481150129L;
```

Comment:
To clarify, you need to capture the `byte[]` and then copy and paste the bytes into your test source / resources